articles/ros2_security_enclaves.md

```diff
@@ -0,0 +1,315 @@
+---
+layout: default
+title: ROS 2 Security Enclaves
+permalink: articles/ros2_security_enclaves.html
+abstract:
+  This article specifies the integration of security enclaves.
+author:  >
+  [Ruffin White](https://github.com/ruffsl),
+  [Mikael Arguedas](https://github.com/mikaelarguedas)
+
+published: true
+categories: Security
+---
+
+{:toc}
+
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+This design document formalizes the integration of ROS 2 with security enclaves.
+In summary, all processes must load a common enclave that contains the runtime security artifacts unique to that enclave, yet each process may not necessarily have a unique enclave.
```

Comment:
all processes -> all secure processes
must load a common enclave -> must "belong to"/"point to"/"declare" (not sure) ? they don't really "load" an enclave per se

Reply:
Done: 8f7fac9